apps/AnnotateVariants/Program.cs

```diff
@@ -40,35 +40,45 @@ static void ProcessCases(List<ASETools.Case> casesToProcess, ASETools.ASEConfirg
 
                 var annotatedVariants = new List<ASETools.AnnotatedVariant>();
 
-                var mafLines = ASETools.MAFLine.ReadFile(case_.extracted_maf_lines_filename, case_.maf_file_id, false);
-
-                if (null == mafLines)
-                {
-                    Console.WriteLine("Case " + case_.case_id + " failed to load extracted MAF lines.  Ignoring.");
-                    continue;
-                }
-
-                foreach (var mafLine in mafLines)
-                {
-                    annotatedVariants.Add(AnnotateVariant(true, case_, mafLine.Chromosome, mafLine.Start_Position, mafLine.Match_Norm_Seq_Allele1, mafLine.Tumor_Seq_Allele2, mafLine.Variant_Type, mafLine.getExtractedReadsExtension()));
-                }
-
-                var selectedVariants = ASETools.SelectedVariant.LoadFromFile(case_.selected_variants_filename);
-                if (null == selectedVariants)
-                {
-                    Console.WriteLine("Case " + case_.case_id + " failed to load selected variants.  Ignoring.");
-                    continue;
-                }
-
-                foreach (var selectedVariant in selectedVariants)
-                {
-                    annotatedVariants.Add(AnnotateVariant(false, case_, selectedVariant.contig, selectedVariant.locus, Convert.ToString(selectedVariant.referenceBase), Convert.ToString(selectedVariant.altBase), "SNP",
-                        selectedVariant.getExtractedReadsExtension()));   // All of the variants we selected are SNPs, so it's just a constant
-                }
-
+				if (somatic)
```

Comment:
where do we want to keep the somatic vs germline values? Should they be saved in the same file?